### PR TITLE
[Identity] UX tweak for id/addr

### PR DESCRIPTION
**Pull request by ccen-stripe**
## Summary

* Change the color of button background from blur to systembackground
* Change the date picker to a text field with date formatters
* Correctly adjust the bottom inset of scroll view when keyboard is hiding
* Add `onDrag` mode for keyboard dismiss
* Change the font of primary button in dark mode
* Change the font&padding of 'My country is not listed'

## Motivation

Improve UX

## Testing

- [x] Manually verified

# Screenshots
| | Before  | After |
| ------------- | ------------- | ------------- |
| button background, date, bottom inset |fixBefore  |fixAfter|
| dismiss keyboard onDrag | dragBefore | dragAfter |
| primary button disable state | disableBefore2  |  disableAfter2  |
| country not listed button font&padding |  buttonBefore | buttonAfter |

### CODE REVIEW

File: StripeIdentity/StripeIdentity/Source/Helpers/String+Localized.swift [hunk 1]
```diff
@@ -20,11 +20,18 @@ extension String.Localized {
...
-            "Date of birth",
+            "Date of Birth",
             "Label for Date of birth field"
         )
     }
 
+    static var date_of_birth_invalid: String {
+        STPLocalizedString(
+            "Date of birth does not look valid",
+            "Message for invalid Date of birth field"
```
Comment (sgrant-stripe) [inline StripeIdentity/StripeIdentity/Source/Helpers/String+Localized.swift:31]: Should those also be capitalized? `Date of Birth`?

File: StripeIdentity/StripeIdentityTests/Snapshot/BiometricConsentViewControllerSnapshotTest.swift [hunk 2]
```diff
@@ -17,7 +17,7 @@ final class BiometricConsentViewControllerSnapshotTest: FBSnapshotTestCase {
     override func setUp() {
         super.setUp()
 
-        //        recordMode = true
+//                recordMode = true
```
Comment (sgrant-stripe) [inline StripeIdentity/StripeIdentityTests/Snapshot/BiometricConsentViewControllerSnapshotTest.swift:20]: Should we just delete this line?

**Review by sgrant-stripe — APPROVED** [review]
My only comment is that I am not a fan of the phone number dial pad being used to enter a date of birth.

File: StripeIdentity/StripeIdentity/Source/Helpers/String+Localized.swift (hunk 1 above)
Comment (ccen-stripe) [reply StripeIdentity/StripeIdentity/Source/Helpers/String+Localized.swift:31]: this is in line with android text, only the section title is capitalized

File: StripeIdentity/StripeIdentityTests/Snapshot/BiometricConsentViewControllerSnapshotTest.swift (hunk 2 above)
Comment (ccen-stripe) [reply StripeIdentity/StripeIdentityTests/Snapshot/BiometricConsentViewControllerSnapshotTest.swift:20]: we use this pattern for other snapshot test as well - https://confluence.corp.stripe.com/display/MOBILE/Snapshot+Tests

> My only comment is that I am not a fan of the phone number dial pad being used to enter a date of birth.

hmm we're using `asciiCapableNumberPad`, it seems to be the value used for other similar fields(e.g the expiration date of a credit card)

**Pull request merged**